devstack/local.conf.nfp

```diff
@@ -49,6 +49,8 @@ if [[ $DEVSTACK_MODE = enterprise ]]; then
 
     # Configure docker images local repository
     DOCKER_IMAGES_URL=http://192.168.100.50/docker_images/
+    
```

Comment:
We should think about how do we submit this to opensource as this local.conf contains enterprise configuration also.